src/api_mocks/postmark.rs

```diff
@@ -1,7 +1,7 @@
 use serde_json::json;
 use trillium::Handler;
-use trillium_api::Json;
+use trillium_api::{Halt, Json};
 use trillium_router::router;
 pub fn mock() -> impl Handler {
-    router().post("/email/withTemplate", Json(json!({})))
+    (router().post("/email/withTemplate", Json(json!({}))), Halt)
```

Comment:
Just to make sure I understand: you're adding `Halt` to these handlers because otherwise requests to paths without handlers yield no response at all, right?

Reply:
Because of how we've set up our mocks, any route that is not handled by postmark or auth0 falls through to the aggregator api. We don't actually want to fall through in circumstances where the origin matches postmark or auth0 but no specific route matches, so we halt. This may be better handled in the origin router, but this was the least-invasive change, as this was encountered incidentally in the creation of this PR